s3torchconnector/src/s3torchconnector/_s3client/_s3client.py

```diff
@@ -28,8 +28,9 @@ def _identity(obj: Any) -> Any:
 
 
 class S3Client:
-    def __init__(self, region: str):
+    def __init__(self, region: str, endpoint: str = ""):
```

Comment:
Can we change all instances of endpoint default to be `None` rather than empty string?